modules/navier_stokes/doc/content/modules/navier_stokes/index.md

```diff
@@ -9,7 +9,8 @@ For documentation specific to finite element or finite volume implementations,
 please refer to the below pages:
 
 - [Continuous Galerkin Finite Element](navier_stokes/cgfe.md)
-- [Finite Volume](navier_stokes/fv.md)
+- [Incompressible Finite Volume](navier_stokes/fv.md)
+- [Porous media Incompressible Finite Volume](navier_stokes/pinsfv.md)
```

Comment:
If you want to do this and your change above (with then changing `###` to `##`), then I think you should `git mv fv.md insfv.md`

Reply:
I would be supportive of that

Reply:
This is what things look like in the `cnsfv-rebased` branch: [index.md](https://github.com/lindsayad/moose/blob/cnsfv-rebased/modules/navier_stokes/doc/content/modules/navier_stokes/index.md) and [fv.md](https://github.com/lindsayad/moose/blob/cnsfv-rebased/modules/navier_stokes/doc/content/modules/navier_stokes/fv.md)

Reply:
I'm going to add the link to `pinsfv.md` to `fv.md` right now as I'm doing the tabular breakdown of NS capabilities

Reply:
Actually yea I like the cutout of the middle-man `fv.md` and putting the links to `insfv.md, cnsfv.md, pinsfv.md` directly in `index.md` and then that allows us to do a tabular summary in `index.md` that includes both FE and FV

Reply:
Ok I ll move fv.md to insfv.md and add that to the PR

Reply:
umm you did it in CNSFV so I guess no need for me to do it

Reply:
Ah I still need to change the ### to ##.
I ll do it soon